packages/client-twitter/src/index.ts

```diff
@@ -34,12 +34,15 @@ export const TwitterClientInterface: Client = {
 
         elizaLogger.log("Twitter client started");
 
-        const manager = new TwitterManager(runtime, this.enableSearch);
+        const manager = new TwitterManager(runtime, runtime.getSetting("TWITTER_SEARCH_ENABLE") === "true");
```

Comment:
you should do a case insensitive comparison so lowercasing the returned setting

Reply:
use toLowerCase() method on 510877303f244f2b0c6afdb8e350f8b0fb4606b0